app/src/pages/Edit.tsx

```diff
@@ -0,0 +1,214 @@
+import { LoaderFunction, useBlocker } from "react-router";
+import { atom } from "jotai";
+import { branchExists, createPr, repoRaw, request } from "@/api";
+import { getAtom, setAtom } from "@/App";
+import { calcCompletion, convert, isEdited, revert } from "@/data/data";
+import lessons from "@/data/lessons.json";
+import { _Entry, _Timings, Entry } from "@/data/types";
+import Footer from "@/pages/edit/Footer";
+import Header from "@/pages/edit/Header";
+import Section from "@/pages/edit/Section";
+
+/** translation edit page */
+function Edit() {
+  /** protect users from losing work */
+  useBlocker(() => !window.confirm(navMessage));
+
+  return (
+    <>
+      <Header />
+      <main>
+        <Section label="title" entries={title} />
+        <Section label="captions" entries={captions} />
+        <Section label="description" entries={description} />
+      </main>
+      <Footer />
+    </>
+  );
+}
+
+export default Edit;
+
+/** page-wide state */
```

Comment:
Probably want to take a glance at this file, from this point down, to be familiar with what's going on.